tests/saw/spec/handshake/handshake_io_lowlevel.saw

```diff
@@ -257,6 +275,7 @@ let s2n_conn_set_handshake_type_spec = do {
     let conn' = {{ conn_set_handshake_type conn }};
     crucible_ghost_value corked {{ conn'.corked }};
     crucible_points_to (conn_handshake_handshake_type pconn) (crucible_term {{ conn'.handshake.handshake_type }});
+    crucible_points_to (conn_handshake_hello_retry_request pconn) (crucible_term {{ conn'.handshake.hello_retry_request }});
```

Comment:
Is it possible here to verify that the hello_retry_request flag is not set in the handshake type when using TLS1.2?

Reply:
Probably? Will need some help with this.